_posts/2021-04-21-frequency-band-control-in-unetstack.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+comments: true
+title: Frequency band control using UnetStack
+date: 21/04/2021
+author: Shiraz S
+categories: howto
+feature-img: "assets/img/freqBandControl/title.png"
+thumbnail: "assets/img/freqBandControl/title.png"
+tags: [howto, unetstack, modems, phy, frequency, band, fdma]
+---
+
+## The basics
+
+An underwater acoustic modem essentially sends out sound signals underwater to transmit information. The typical transmit processing takes in user's binary data and after error correction encoding, modulates a carrier according to some modulation scheme such as OFDM or FHBFSK, the details of which the reader can understand from other sources. 
+
+For standard medium frequency Subnero modems, the fundamental centre frequency `C=24kHz`.
+The baseband complex signals are clocked out via the DAC (Digital to Analog Converter) at the rate of `R=24000 samples/s`, which translates to a total bandwidth of `B=24kHz` (we shall omit the details of wireless communication theory in this blog). So the band goes from 12kHz to 36kHz.
```

Comment:
We advertise our bandwidth as 20 - 32 kHz instead of 36 kHz. Would it be possible to differentiate the theoretical bandwidth and what the modem actually uses (from a hardware standpoint)? Or is this differentiation irrelevant?

Reply:
It would be good to mention that not all of this bandwidth is supported by the transducer, and so a modem's usable bandwidth is typically smaller (and give M25M data sheet value here)

Reply:
made changes to address this